schemas/compat-data.schema.json

```diff
@@ -43,8 +43,20 @@
           "type": "boolean",
           "description": "A boolean value indicating whether or not the implementation of the sub-feature deviates from the specification in a way that may cause compatibility problems. It defaults to false (no interoperability problems expected). If set to true, it is recommended that you add a note explaining how it diverges from the standard (such as that it implements an old version of the standard, for example)."
         },
-        "version_added": { "$ref": "#/definitions/version_value" },
-        "version_removed": { "$ref": "#/definitions/version_value" },
+        "version_added": {
+          "description": "A string the value true, indicating which browser version added this feature, or the value false indicating the feature is not supported, or the value null indicating support is unknown.",
+          "allOf": [
+            { "type": ["string", "boolean", "null"] },
+            { "not": { "enum": ["true", "false", "null"] } }
+          ]
+        },
+        "version_removed": {
+          "description": "A string or the value true, indicating which browser version removed this feature.",
+          "allOf": [
+            { "type": ["string", "boolean"] },
+            { "not": { "enum": ["true", false] } }
```

Comment:
This is not the most obvious way of saying that true is an allowed value, but convenient since the string "true" is already excluded here.